cmd/mountv1_test.go

```diff
@@ -441,13 +441,17 @@ func (suite *generateConfigTestSuite) TestConfigCLILogLevel() {
 func (suite *generateConfigTestSuite) TestCLIParamLogging() {
 	defer suite.cleanupTest()
 	name := generateFileName()
+	v1ConfigFile, _ := ioutil.TempFile("", name+".tmp.cfg")
+	defer os.Remove(v1ConfigFile.Name())
+	v1ConfigFile.WriteString("accountName myAccountName")
```

Comment:
Are we trying to force put the account name in config file? If in v1 it was given through env-var then it shall go as env-var to v2 as well.